### ISSUE TITLE
Error on google-cloud-firebase gem startup

### ISSUE THREAD

**Issue by xdougx** [googleapis/google-cloud-ruby#25741]
#### Environment details

  - OS:  ManjaroLinux 23.1.4
  - Ruby version: 3.0.6
  - Gem name and version: google-cloud-firestore v 2.10.0

#### Steps to reproduce

  1. Installed in gemfile
  
  ```
# GCloud
gem 'google-cloud-storage'
gem 'google-cloud-bigquery'
gem 'google-cloud-pubsub'
gem 'google-cloud-firestore'
  ```
require in rails application.rb

```
require "google/cloud/storage"
require "google/cloud/bigquery"
require "google/cloud/pubsub"
require "google/cloud/firestore"
```

#### Full backtrace

```sh
 douglas@douglas  service-content-blueprint   master ±  rails c
/home/douglas/.rbenv/versions/3.0.6/lib/ruby/gems/3.0.0/gems/google-cloud-firestore-2.10.0/lib/google/cloud/firestore/aggregate_query.rb:172:in `<class:AggregateQuery>': uninitialized constant Google::Cloud::Firestore::V1::StructuredAggregationQuery (NameError)
Did you mean?  Google::Cloud::Firestore::V1::StructuredQuery
	from /home/douglas/.rbenv/versions/3.0.6/lib/ruby/gems/3.0.0/gems/google-cloud-firestore-2.10.0/lib/google/cloud/firestore/aggregate_query.rb:60:in `<module:Firestore>'
	from /home/douglas/.rbenv/versions/3.0.6/lib/ruby/gems/3.0.0/gems/google-cloud-firestore-2.10.0/lib/google/cloud/firestore/aggregate_query.rb:20:in `<module:Cloud>'
	from /home/douglas/.rbenv/versions/3.0.6/lib/ruby/gems/3.0.0/gems/google-cloud-firestore-2.10.0/lib/google/cloud/firestore/aggregate_query.rb:19:in `<module:Google>'
	from /home/douglas/.rbenv/versions/3.0.6/lib/ruby/gems/3.0.0/gems/google-cloud-firestore-2.10.0/lib/google/cloud/firestore/aggregate_query.rb:18:in `<main>'
	from /home/douglas/.rbenv/versions/3.0.6/lib/ruby/gems/3.0.0/gems/bootsnap-1.17.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:32:in `require'
	from /home/douglas/.rbenv/versions/3.0.6/lib/ruby/gems/3.0.0/gems/bootsnap-1.17.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:32:in `require'
	from /home/douglas/.rbenv/versions/3.0.6/lib/ruby/gems/3.0.0/gems/google-cloud-firestore-2.10.0/lib/google/cloud/firestore/query.rb:20:in `<main>'
	from /home/douglas/.rbenv/versions/3.0.6/lib/ruby/gems/3.0.0/gems/bootsnap-1.17.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:32:in `require'
	from /home/douglas/.rbenv/versions/3.0.6/lib/ruby/gems/3.0.0/gems/bootsnap-1.17.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:32:in `require'
	from /home/douglas/.rbenv/versions/3.0.6/lib/ruby/gems/3.0.0/gems/google-cloud-firestore-2.10.0/lib/google/cloud/firestore/collection_reference.rb:17:in `<main>'
	from /home/douglas/.rbenv/versions/3.0.6/lib/ruby/gems/3.0.0/gems/bootsnap-1.17.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:32:in `require'
	from /home/douglas/.rbenv/versions/3.0.6/lib/ruby/gems/3.0.0/gems/bootsnap-1.17.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:32:in `require'
	from /home/douglas/.rbenv/versions/3.0.6/lib/ruby/gems/3.0.0/gems/google-cloud-firestore-2.10.0/lib/google/cloud/firestore/client.rb:20:in `<main>'
	from /home/douglas/.rbenv/versions/3.0.6/lib/ruby/gems/3.0.0/gems/bootsnap-1.17.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:32:in `require'
	from /home/douglas/.rbenv/versions/3.0.6/lib/ruby/gems/3.0.0/gems/bootsnap-1.17.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:32:in `require'
	from /home/douglas/.rbenv/versions/3.0.6/lib/ruby/gems/3.0.0/gems/google-cloud-firestore-2.10.0/lib/google/cloud/firestore.rb:17:in `<main>'
	from /home/douglas/.rbenv/versions/3.0.6/lib/ruby/gems/3.0.0/gems/bootsnap-1.17.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:32:in `require'
	from /home/douglas/.rbenv/versions/3.0.6/lib/ruby/gems/3.0.0/gems/bootsnap-1.17.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:32:in `require'
	from /home/douglas/workspace/ahazou/services/service-content-blueprint/config/application.rb:17:in `<main>'
	from /home/douglas/.rbenv/versions/3.0.6/lib/ruby/gems/3.0.0/gems/bootsnap-1.17.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:32:in `require'
	from /home/douglas/.rbenv/versions/3.0.6/lib/ruby/gems/3.0.0/gems/bootsnap-1.17.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:32:in `require'
	from /home/douglas/.rbenv/versions/3.0.6/lib/ruby/gems/3.0.0/gems/railties-6.1.7.6/lib/rails/command/actions.rb:22:in `require_application!'
	from /home/douglas/.rbenv/versions/3.0.6/lib/ruby/gems/3.0.0/gems/railties-6.1.7.6/lib/rails/command/actions.rb:14:in `require_application_and_environment!'
	from /home/douglas/.rbenv/versions/3.0.6/lib/ruby/gems/3.0.0/gems/railties-6.1.7.6/lib/rails/commands/console/console_command.rb:101:in `perform'
	from /home/douglas/.rbenv/versions/3.0.6/lib/ruby/gems/3.0.0/gems/thor-1.3.0/lib/thor/command.rb:28:in `run'
	from /home/douglas/.rbenv/versions/3.0.6/lib/ruby/gems/3.0.0/gems/thor-1.3.0/lib/thor/invocation.rb:127:in `invoke_command'
	from /home/douglas/.rbenv/versions/3.0.6/lib/ruby/gems/3.0.0/gems/thor-1.3.0/lib/thor.rb:527:in `dispatch'
	from /home/douglas/.rbenv/versions/3.0.6/lib/ruby/gems/3.0.0/gems/railties-6.1.7.6/lib/rails/command/base.rb:69:in `perform'
	from /home/douglas/.rbenv/versions/3.0.6/lib/ruby/gems/3.0.0/gems/railties-6.1.7.6/lib/rails/command.rb:48:in `invoke'
	from /home/douglas/.rbenv/versions/3.0.6/lib/ruby/gems/3.0.0/gems/railties-6.1.7.6/lib/rails/commands.rb:18:in `<main>'
	from /home/douglas/.rbenv/versions/3.0.6/lib/ruby/gems/3.0.0/gems/bootsnap-1.17.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:32:in `require'
	from /home/douglas/.rbenv/versions/3.0.6/lib/ruby/gems/3.0.0/gems/bootsnap-1.17.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:32:in `require'
	from bin/rails:4:in `<main>'

```

Thanks!

**Comment by xdougx** [comment]
Hi, seem i fixed setting up the versions manually

```
gem 'google-cloud-storage', '~> 1.50.0'
gem 'google-cloud-bigquery', '~> 1.49.0'
gem 'google-cloud-pubsub', '~> 2.18.0'
gem 'google-cloud-firestore', '~> 2.15.0'
```
but i dont know if is still a issue for whom goes without version in gemfile

**Comment by aandreassa** [comment]
Hi @xdougx, this was a similar issue to #20354. 

The problem was fixed in [v2.10.1](https://github.com/googleapis/google-cloud-ruby/releases/tag/google-cloud-firestore%2Fv2.10.1) a while back. 

Unfortunately, based on the bug report you were using `v2.10.0`, so you just barely missed that patch! PLMK if you run into similar issues. Thanks!
